### PR TITLE
[DEV-4191] Fix mapview to not show the Web campus

### PR DESCRIPTION
**Pull request by richarddubay**
### What does this PR do, or why is it needed?
This PR removed the Web campus from the map view.

### How do I test this PR?
With your campus set as "Web" and not as "Web", go to the locations page and make sure that all you see are the 14 regular campuses.

## TODO

- [x] I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))
- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [x] No new warnings in tests, in storybook, and in-app
- [x] Upload GIF(s) of iOS and Android if applicable
- [x] Set a relevant reviewer

## REVIEW

- [ ] Review updates to test coverage and snapshots
- [ ] Review code through the lens of being concise, simple, and well-documented

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by redreceipt** [comment]
could we just filter it out in the `sortedCampuses` function?

**Comment by richarddubay** [comment]
@redreceipt Okay, changed that.

File: packages/newspringchurchapp/src/user-settings/Locations/MapView.js [hunk 1]
```diff
@@ -102,11 +102,13 @@ class MapView extends Component {
   get sortedCampuses() {
     const { currentCampus = null, campuses = [] } = this.props;
```
Comment (redreceipt) [inline packages/newspringchurchapp/src/user-settings/Locations/MapView.js:103]:
```suggestion
    const { currentCampus = null, campuses = [] } = this.props;
    const publicCampuses = campuses.filter(({ name }) => name !== 'Web');
```

File: packages/newspringchurchapp/src/user-settings/Locations/MapView.js [hunk 2]
```diff
@@ -102,11 +102,13 @@ class MapView extends Component {
...
     if (!this.props.currentCampus) {
-      return campuses;
+      return campuses.filter(({ name }) => name !== 'Web');
```
Comment (redreceipt) [inline packages/newspringchurchapp/src/user-settings/Locations/MapView.js]:
```suggestion
      return publicCampuses;
```

File: packages/newspringchurchapp/src/user-settings/Locations/MapView.js [hunk 3]
```diff
@@ -102,11 +102,13 @@ class MapView extends Component {
...
     }
     return [
       currentCampus,
-      ...campuses.filter(({ id }) => id !== currentCampus.id),
+      ...campuses
```
Comment (redreceipt) [inline packages/newspringchurchapp/src/user-settings/Locations/MapView.js]:
```suggestion
      ...publicCampuses
```

File: packages/newspringchurchapp/src/user-settings/Locations/MapView.js [hunk 4]
```diff
@@ -102,11 +102,13 @@ class MapView extends Component {
...
+        .filter(({ id }) => id !== currentCampus.id)
+        .filter(({ name }) => name !== 'Web'),
```
Comment (redreceipt) [inline packages/newspringchurchapp/src/user-settings/Locations/MapView.js]:
```suggestion
```

**Comment by redreceipt** [comment]
sorry, i'm being nitpicky. left a couple more comments to remove redundancy.

**Review by redreceipt — DISMISSED** [review]

**Comment by richarddubay** [comment]
@redreceipt Now when I'm testing this I am getting the Web campus to show up ... I think because `currentCampus` is Web. Not sure why I didn't see that before.

**Comment by richarddubay** [comment]
@redreceipt I feel like this is much better and covers all our bases.

**Review by redreceipt — APPROVED** [review]

**Pull request merged**